app/models/thredded/post.rb

```diff
@@ -23,7 +23,7 @@ def private_topic_post?
 
     # @return [ActiveRecord::Relation<Thredded.user_class>] users from the list of user names that can read this post.
     def readers_from_user_names(user_names)
-      CaseInsensitiveStringFinder
+      DbTextSearch::CaseInsensitiveEq
         .new(Thredded.user_class.thredded_messageboards_readers([messageboard]), Thredded.user_name_column)
```

Comment:
I know that's not part of the diff but this *just* caught my eye due to proximity :)